src/redux/books/bookSlice.js

```diff
@@ -2,24 +2,45 @@
 import { createSlice } from '@reduxjs/toolkit';
 
 const initialState = {
-  books: [],
-  isLoading: false,
-  error: undefined,
+  bookItems: [
+    // Initial state:
+    {
+      item_id: 'item1',
+      title: 'The Great Gatsby',
+      author: 'John Smith',
+      category: 'Fiction',
+    },
+    {
+      item_id: 'item2',
+      title: 'Anna Karenina',
+      author: 'Leo Tolstoy',
+      category: 'Fiction',
+    },
+    {
+      item_id: 'item3',
+      title: 'The Selfish Gene',
+      author: 'Richard Dawkins',
+      category: 'Nonfiction',
+    },
+  ],
```

Comment:
- [ ] These initial books are not listed on the page when I load the page. Kindly fix that.

  ![image](https://github.com/evansnyamekye/bookstore/assets/36542986/76113000-9867-41dd-839a-72675cbfdeba)